bindings/java/src/org/sleuthkit/datamodel/HostManager.java

```diff
@@ -51,6 +51,54 @@
 		this.db = skCase;
 	}
 	
+	/**
+	 * APTODO : using for testing - might be able to remove if desired
```

Comment:
I think you could probably take this comment out.  I think this method will be useful in autopsy.

Reply:
Yes I think so - I used it in the temporary DSP code, and I'm pretty sure you'll need to do the same.